xbmc/utils/PlayerUtils.cpp

```diff
@@ -31,7 +31,7 @@ bool CPlayerUtils::IsItemPlayable(const CFileItem& itemIn)
     return true;
 
   // Movies / TV Shows / Music Videos
-  if (VIDEO_UTILS::IsItemPlayable(item))
+  if (IsItemPlayable(item))
```

Comment:
Doesn't this now call itself recursively? I think the namespace needs to be kept to call the right function.

Reply:
indeed.